src/BlazorUI/Bit.BlazorUI/Components/Utilities/Label/BitLabel.scss

```
@@ -14,13 +14,67 @@
 
     &.bit-dis {
         color: $clr-fg-dis;
+        // The label itself is never disabled - only the control it names is - so the pointer over it would still
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)